lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
I guess if you keep the `MakeExecutable()` definition here and move the calls to it into the `package.py` files, it won't be that much duplicated code. A big benefit of that would be that it would be impossible to call `cmake()` without adding `depends_on('cmake', type='build')`. But you wouldn't be able to move `make()` or `gmake()`...